cypress/integration/authentication.spec.js

```diff
@@ -129,7 +129,7 @@ describe("password verification", () => {
       form: true,
     });
     cy.visit("/members"); // Check login actually worked once
-    cy.request("POST", "/auth/logout");
```

Comment:
This was to fix a test that looked broken in the videos, but wasn't actually an issue